packages/devtools_app/lib/src/debugger/flutter/codeview.dart

```diff
@@ -241,26 +250,31 @@ class GutterRow extends StatelessWidget {
                 child: SizedBox(
                   width: bpBoxSize,
                   height: bpBoxSize,
-                  child: Stack(
-                    alignment: AlignmentDirectional.center,
-                    children: [
-                      if (isExecutable)
-                        createCircleWidget(executableLineRadius, subtleColor),
-                      if (isBreakpoint)
-                        createCircleWidget(breakpointRadius, foregroundColor),
-                    ],
+                  child: Center(
+                    child: createAnimatedCircleWidget(
+                      isBreakpoint ? breakpointRadius : executableLineRadius,
+                      isBreakpoint ? foregroundColor : subtleColor,
+                    ),
                   ),
                 ),
               ),
             Text('$lineNumber', textAlign: TextAlign.end),
-            if (isPausedHere)
-              const Padding(
-                padding: EdgeInsets.only(left: executionPointIndent),
-                child: Align(
-                  alignment: Alignment.centerLeft,
-                  child: Icon(Icons.play_arrow, size: defaultIconSize),
+            Padding(
```

Comment:
nit: combine the Padding + Align into a Container widget.

Reply:
will do

Reply:
cc @scheglov, for some possible flutter related refactorings (convert Padding to Container; convert Align to Container; combine Padding with parent Container, ...)